project/assembly.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.6")
```

Comment:
sbt-assembly

Reply:
Hi @junqueira, can you explain why we should have sbt-assembly added to this project?

Reply:
I believe it is the default wheel "sbt assembly" and everything works as expected, or am I wrong?